docs/source/start-routingspec.rst

```diff
@@ -662,22 +717,200 @@ so consider the name before saving.
 
 Once the new window comes up the user should address the following fields, at least.
 
-+-------------------------+---------------------------------------------------+
-|**Field**                |**Description**                                    |
-+=========================+===================================================+
-|Data Source              |Location in Database                               |
-+-------------------------+---------------------------------------------------+
-|Destination              |consumer (pipe, database, file, etc)               |
-+-------------------------+---------------------------------------------------+
-|Output Format            |set output format                                  |
-+-------------------------+---------------------------------------------------+
-|Date/Time - Since/Util   |define time window for message retreival           |
-+-------------------------+---------------------------------------------------+
-|Platform Selection       |Specify a platform or network list                 |
-+-------------------------+---------------------------------------------------+
++-------------------------+------------------------------------------------+
+|**Field**                |**Description**                                 |
++=========================+================================================+
+|Data Source              |Location in Database                            |
++-------------------------+------------------------------------------------+
+|Destination              |consumer (pipe, database, file, etc)            |
++-------------------------+------------------------------------------------+
+|Output Format            |set output format                               |
++-------------------------+------------------------------------------------+
+|Date/Time - Since/Util   |define time window for message retrieval        |
++-------------------------+------------------------------------------------+
+|Platform Selection       |Specify a platform or network list              |
++-------------------------+------------------------------------------------+
+
```

Comment:
The first sentence is rendering as a Title.
![image](https://github.com/opendcs/opendcs/assets/4818531/7d45086d-26dc-49be-a75f-ed3407bdedd7)

Reply:
Right now it's looking reasonable format on my end - let me know if this is still showing up goofy? 
![image](https://github.com/opendcs/opendcs/assets/104451600/33f260c3-33ef-432d-ab2b-f3db6b5e624a)